### PR TITLE
Style: Run scalafmt on the whole repo

### PR DESCRIPTION
**Pull request by j-hall-mwam**
Standardize code style for entire repository via scalafmt.

### CODE REVIEW

**Review by mmniazi — APPROVED** [review]
LGTM

File: kafkakewl-common/src/main/scala/com/mwam/kafkakewl/common/http/EndpointUtils.scala [hunk 1]
```diff
@@ -13,36 +13,48 @@ import sttp.tapir.EndpointIO.annotations.*
 import zio.json.{JsonDecoder, JsonEncoder}
 
 trait EndpointUtils {
-  val apiEndpoint: PublicEndpoint[Unit, Unit, Unit, Any] = endpoint.in("api" / "v1")
+  val apiEndpoint: PublicEndpoint[Unit, Unit, Unit, Any] =
+    endpoint.in("api" / "v1")
```
Comment (mw-lb) [inline kafkakewl-common/src/main/scala/com/mwam/kafkakewl/common/http/EndpointUtils.scala:17]: By default `maxColumn = 80`, should we use something larger? I'd say at least 120, but I'm fine with more.

(I'm not a fan of too many line-breaks, less code fits in the screen)

Comment (j-hall-mwam) [reply kafkakewl-common/src/main/scala/com/mwam/kafkakewl/common/http/EndpointUtils.scala]: Done

**Review by mw-lb — DISMISSED** [review]
Looks better, we'll see how  much those newlines annoy me and possibly increase the maxColumn further :)

**Review by mw-lb — APPROVED** [review]

**Pull request merged**